src/org/thoughtcrime/securesms/ConnectivityActivity.java

```diff
@@ -35,7 +35,8 @@ public boolean onPrepareOptionsMenu(Menu menu) {
   }
 
   private void refresh() {
-    String connectivityHtml = DcHelper.getContext(this).getConnectivityHtml();
+    final String connectivityHtml = DcHelper.getContext(this).getConnectivityHtml()
+                                      .replace("</style>", " html { color-scheme: dark light; }</style>");
```

Comment:
this is a bit hackish, but a similar approach as on desktop/ios

Reply:
I guess this should be added to core sooner or later